notebooks/SITCOMTN_101_FCU_investigation.ipynb

```diff
@@ -0,0 +1,1216 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        percentv,begin, end = fcu_times.iloc[i,:]
</pre><p>Can you add an space between <code>percentv</code> and <code>begin</code> , right after the comma? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-101/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>